integration/testdata/conda-cyclonedx.json.golden

```diff
@@ -27,7 +27,7 @@
   },
   "components": [
     {
-      "bom-ref": "pkg:conda/openssl@1.1.1q?file_path=miniconda3%2Fenvs%2Ftestenv%2Fconda-meta%2Fopenssl-1.1.1q-h7f8727e_0.json",
+      "bom-ref": "pkg:conda/openssl@1.1.1q",
```

Comment:
We need the file path in BOM-Ref. While BOM-Ref must be unique, PURLs can conflict. This is another example where BOM-Ref is different from PURL.

Reply:
fixed at https://github.com/aquasecurity/trivy/pull/5439/commits/4a83a34af3e1ab3ade1380ee21ac74b8f59306a9